test/visual/visual.js

```diff
@@ -60,6 +60,7 @@ module.exports = {
                     userDataDir: `${baselineDir}/${type}/userDataDir`,
                 });
                 page = await browser.newPage();
+                await page.mouse.move(-5, -5);
```

Comment:
Attempts to ensure we're never "hovering" over anything when the screenshot occurs.